report_py3o/models/py3o_report.py

```diff
@@ -64,12 +65,22 @@ def fct1(fct):
     return fct1
 
 
+def format_multiline_value(value):
+    if value:
+        return Markup(unicode(value).replace('<', '&lt;').replace('>', '&gt;').
```

Comment:
@omar7r Why do you need to call 'unicode'??? What if the value is not unicode compliant?

Reply:
It's not needed. I just deleted it.